### PR TITLE
Introducing the Override Bot

### PR DESCRIPTION
**Pull request by orenc1**
Signed-off-by: orenc1 <ocohen@redhat.com>

On first step, the override bot can invoked by commenting `/override-bot` on any open PR.
Then it will comment `/override ci/prow/...` for each failed/pending lane that has a parallel passed lane on another cloud provider, for all open PRs in this repository.

**Release note**:
```release-note
Override Bot implementation, using github-actions and python.
```

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

File: automation/override-bot/override-bot.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        statuses = json.loads(statuses_raw)
+        for status in statuses:
+            context = status['context']
+            if 'ci-index' in context or 'images' in context or 'prow' not in context:
+                continue
+            splitted = context.split('/')[-1].split('-')
+            provider = splitted[-1]
+            test_name = '-'.join(splitted[:-1])
+            state = status['state']
+            overridden = True if status['description'] and 'Overridden' in status['description'] else False
```
Comment (nunnatsa) [inline automation/override-bot/override-bot.py:69]: A Boolean value is just a result of a Bolean expression. no need to explicitly set True and False; e.g.
overridden = status['description'] and 'Overridden' in status['description']
```python

```

**Review by nunnatsa — APPROVED** [review]

File: automation/override-bot/override-bot.py (hunk 1 above)
Comment (orenc1) [reply automation/override-bot/override-bot.py:69]: Thanks for pointing that out. I'll change in a following PR.

**Pull request merged**